homework3/hw3/Views/MatchView.swift

```diff
@@ -0,0 +1,154 @@
+        awayTeamLabel.lineBreakMode = .byWordWrapping
+        
+        homeScoreLabel.setStyle(.regular14)
+        homeScoreLabel.textAlignment = .right
+        awayScoreLabel.setStyle(.regular14)
+        awayScoreLabel.textAlignment = .right
+        
+    }
+    
+    override func setupConstraints() {
+        super.setupConstraints()
+                
+        timeLabel.snp.makeConstraints {
+            $0.leading.equalToSuperview().offset(4)
+            $0.top.equalToSuperview().offset(10)
+            $0.width.equalTo(56)
+        }
+            
+        timeStatusLabel.snp.makeConstraints {
+            $0.centerX.equalTo(timeLabel)
+            $0.top.equalTo(timeLabel.snp.bottom).offset(4)
+            $0.leading.equalToSuperview().inset(16)
+        }
+            
+        dividerView.snp.makeConstraints {
+            $0.leading.equalTo(timeLabel.snp.trailing).offset(16)
+            $0.width.equalTo(1)
+            $0.top.bottom.equalToSuperview().inset(8)
+        }
+            
+        homeTeamLogoImageView.snp.makeConstraints {
+            $0.leading.equalTo(dividerView.snp.trailing).offset(16)
+            $0.size.equalTo(16)
+            $0.top.equalToSuperview().offset(10)
+        }
+            
+        homeTeamLabel.snp.makeConstraints {
+            $0.centerY.equalTo(homeTeamLogoImageView)
+            $0.leading.equalTo(homeTeamLogoImageView.snp.trailing).offset(8)
+            $0.trailing.equalTo(homeScoreLabel.snp.leading).offset(-8)
+        }
+            
+        homeScoreLabel.snp.makeConstraints {
+            $0.centerY.equalTo(homeTeamLogoImageView)
+            $0.trailing.equalToSuperview().offset(-16)
+            $0.leading.greaterThanOrEqualTo(homeTeamLogoImageView.snp.trailing).offset(8)
+        }
+            
+        awayTeamLogoImageView.snp.makeConstraints {
+            $0.top.equalTo(homeTeamLogoImageView.snp.bottom).offset(4)
+            $0.leading.equalTo(dividerView.snp.trailing).offset(16)
+            $0.size.equalTo(16)
+        }
+                
+        awayTeamLabel.snp.makeConstraints {
+            $0.centerY.equalTo(awayTeamLogoImageView)
+            $0.leading.equalTo(awayTeamLogoImageView.snp.trailing).offset(8)
+            $0.trailing.equalTo(awayScoreLabel.snp.leading).offset(-8)
+        }
+                
+        awayScoreLabel.snp.makeConstraints {
+            $0.centerY.equalTo(awayTeamLogoImageView)
+            $0.trailing.equalToSuperview().offset(-16)
+            $0.leading.greaterThanOrEqualTo(awayTeamLogoImageView.snp.trailing).offset(8)
+        }
+    }
+        
+    func configure(with viewModel: MatchViewModel) {
+        let viewModel = viewModel
+        
+        timeLabel.text = viewModel.time
+        timeStatusLabel.text = viewModel.timeStatusText
+        timeStatusLabel.textColor = viewModel.timeStatusColor
+        
+        guard let homeTeamLogo = viewModel.homeTeamLogo else {
+            return
+        }
```

Comment:
Ovo nije dobro, returnat ćeš usred configurea ako nemaš sliku, a može se dogoditi. Na taj način se neće view dobro konfigurirati. Pošalji optinal image unutra. Idealno bi bilo imati neki placeholder ako nema imagea ili da bude postavljan dok se loada async image. Ali ako nema slike, nema slike.